content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,120 @@
+{{% /notes %}}
+
+Type `aws-python` or `aws-typescript` and click **next**.
+
+### Project details
+
+Provide a name for the project, along with an optional description and the stack name which will be set in the resulting `Pulumi.yaml` file and the name of `Pulumi.<stack>.yaml`.
+
+### Configuration
+
+Provide any values required for the configuration declared in the [template](/docs/pulumi-cloud/developer-portals/templates), in this example, you can set the **aws:region** you would like to deploy into.
+
+These values populate the `config` section in the `Pulumi.<stack>.yaml` file, specifying the necessary settings for your project.
+
+### Environment
+
+If you've configured [environments](/docs/pulumi-cloud/esc) with your organization, you can specify one to use with the resulting stack. Environments enable teams to manage hierarchical collections of configuration and secrets without needing to re-specify all of them during project creation.
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end and can be set on each stack. However, by creating an environment with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+### Repository
+
+Configure the repository and optional subdirectory to use when committing your new project code.
+
+{{% notes "info" %}}
+If you granted the Pulumi GitHub app access to _all_ repositories, the New Project Wizard will allow users to create projects with Deployments enabled in new repositories.
+
+If the app only has access to _some_ repositories, users will only be able to create new projects with Deployments enabled in _existing_ repositories.
+{{% /notes %}}
+
+The GitHub owner is not configurable, as that must match the Pulumi GitHub app's owner in order to work with Deployments.
+
+### Deployment settings
+
+Configure the behavior of Deployments, including when to trigger new Deployments and any environment variables required for the Pulumi program to run.
+
+A full description of each setting is available [here](/docs/pulumi-cloud/deployments/reference/#deployment-settings).
+
+### Deployment method
+
+Finally, select Pulumi Deployments as the deployment method and **Create project** to trigger a deployment. You will see your new deployment start within your **Stacks Deployment** tab.
+
+![Animation of how to create a new project in Pulumi](./pulumi-new-project-wizard.gif)
+
+When your deployment is complete, you can now manage and deploy your infrastructure from the Pulumi Cloud Console. In your stack's **Actions** tab, use the **[Click to Deploy](https://www.pulumi.com/docs/pulumi-cloud/deployments/reference/#click-to-deploy)** button to trigger a deployment.
```

Comment:
I'm honestly not sure we should assume the happy path here. I don't know about you, but this almost never works for me on the first try; there's just about always something else that has to be done to get to a green deployment. 

Indeed, as currently written, this tutorial _can't_ work unless the user pastes their AWS credentials into the UI as environment variables at setup time. Some users will be reluctant to do this, but there's really no way for this work without it. We should directly address rather than sidestep this IMO.